tests/integration/test_transcribe.py

```diff
@@ -1,7 +1,127 @@
+import os
+
 import pytest
+from botocore.exceptions import ClientError
+
+from localstack.aws.api.transcribe import BadRequestException, NotFoundException
+from localstack.utils.strings import short_uid
+from localstack.utils.sync import poll_condition
+
+
+@pytest.fixture(autouse=True)
+def transcribe_snapshot_transformer(snapshot):
+    snapshot.add_transformer(snapshot.transform.transcribe_api())
 
 
 class TestTranscribe:
     @pytest.mark.skip_offline
-    def test_transcribe_happy_path(self):
-        pass
+    @pytest.mark.aws_validated
+    @pytest.mark.skip_snapshot_verify(
+        paths=[
+            "$..TranscriptionJob..MediaSampleRateHertz",
+            "$..TranscriptionJob..Settings",
+            "$..Error..Code",
+        ]
+    )
+    def test_transcribe_happy_path(self, transcribe_client, transcribe_create_job, snapshot):
+        job_name = transcribe_create_job(test_file=os.path.abspath("files/en-gb.wav"))
+
+        transcribe_client.get_transcription_job(TranscriptionJobName=job_name)
+
+        def is_transcription_done():
+            transcription_status = transcribe_client.get_transcription_job(
+                TranscriptionJobName=job_name
+            )
+            return transcription_status["TranscriptionJob"]["TranscriptionJobStatus"] == "COMPLETED"
+
+        # empirically it takes around
+        # <5sec for a vosk transcription
```

Comment:
Would this assumption hold if the Vosk language model needs to be downloaded?

Reply:
I cannot check every model, but it does hold for the "en" models (with deleting the cached folder in .filesystem beforehand)... on my setup :man_shrugging: 
I'm happy to change the timeout to any value you think is more appropriate